src/lib/Components/Artist/Header.tsx

```diff
@@ -16,17 +16,18 @@ import SerifText from "../Text/Serif"
 
 const isPad = Dimensions.get("window").width > 700
 
-interface HeaderProps extends React.Props<Header> {
-  artist: any
-}
+// tslint:disable-next-line:no-empty-interface
+interface Props extends RelayProps {}
 
 interface State {
   following: boolean
   followersCount: number
 }
 
+const track: Track<Props, State, Schema.Entity> = _track
```

Comment:
did you consider an exposed HOC with those three generic params that passes it into a blank Component?

Reply:
Can you give a pseudo code example?

Reply:
Any further discussion here or shall we merge? @orta @alloy

Reply:
In case it wasn’t obvious, the first commit used a function call to offer various type overloads and cast the result, whereas the idea of this second commit was to get rid of noop function calls completely.

Reply:
Let's merge, it'd only be a refactor anyway